src/core/Akka.Persistence/Fsm/PersistentFSMBase.cs

```diff
@@ -1059,7 +1059,7 @@ public State ForMax(TimeSpan timeout)
             /// <summary>
             /// TBD
             /// </summary>
-            public List<object> Replies
+            public IReadOnlyCollection<object> Replies
```

Comment:
It shouldn't really affect how end-users are consuming it that much, unless they were modifying the collection themselves which would have been "illegal" anyway. Guess this falls into the gray area of "is fixing a bug in the public API really a breaking change?" I'm fine with it.